app/src/main/java/com/globant/pokemontcgapp/fragment/PokemonTypeFragment.kt

```diff
@@ -32,20 +34,40 @@ class PokemonTypeFragment : Fragment() {
         pokemonTypeViewModel.getPokemonTypes()
     }
 
-    private fun updateUI(pokemonTypes: PokemonTypeData) {
-        when (pokemonTypes.state) {
-            PokemonTypeState.LOADING -> binding.pokemonTypeProgressBar.visibility = View.VISIBLE
-            PokemonTypeState.POKEMON_TYPE_DATA -> pokemonTypes.data?.pokemonTypesList?.let { showPokemonTypes(it) }
+    override fun onPause() {
+        binding.pokemonTypeRecyclerView.visibility = View.GONE
```

Comment:
why do you hide this view in the onPause?

Reply:
It was doing a bad usage of the recycler view, now I removed the method and change the way the recycler and the loader are shown.